docs/getting-started-guides/docker.md

```diff
@@ -0,0 +1,71 @@
+## Running kubernetes locally via Docker
+
+### Step One: Run etcd
+```sh
+docker run --net=host -d kubernetes/etcd:2.0.5.1 /usr/local/bin/etcd --addr=$(hostname -i):4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
+```
+
+### Step Two: Run the master
+```sh
+docker run --net=host -d gcr.io/google-containers/hyperkube:v0.14.1 /hyperkube apiserver --portal_net=10.0.0.1/24 --address=127.0.0.1 --etcd_servers=http://127.0.0.1:4001 --cluster_name=kubernetes --v=2
+```
+
+### Step Three: Run the controller manager
+```sh
+docker run --net=host -d gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube controller-manager --master=127.0.0.1:8080 --machines=127.0.0.1 --sync_nodes=true --v=2
+```
+
+### Step Four: Run the scheduler
+```sh
+docker run --net=host -d gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube scheduler --master=127.0.0.1:8080 --v=2
+```
+
+### Step Five: Run the kubelet
+```sh
+docker run --net=host -d -v /var/run/docker.sock:/var/run/docker.sock  gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube kubelet --api_servers=http://localhost:8080 --v=2 --address=0.0.0.0 --enable_server --hostname_override=127.0.0.1
+```
+
+### Step Six: Run the service proxy
+```sh
+docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube proxy --master=http://127.0.0.1:8080 --v=2
+```
+
+### Test it out
+At this point you should have a running kubernetes cluster.  You can test this by downloading the kubectl 
+binary
+([OS X](http://storage.googleapis.com/kubernetes-release/release/v0.14.1/bin/darwin/amd64/kubectl))
```

Comment:
An alternative could be to have the kubectl read pod from stdin or a volume and launch it with docker run hyperkube kubectl ;)